apps/ledger-live-mobile/src/newArch/features/WalletSync/drawers/addAccount/index.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
If we follow the guidelines we can't add a `drawers` folder here. 
We should put AddAccount (PascalCase) under `screens` or `components` folder.
https://ledgerhq.atlassian.net/wiki/spaces/WXP/pages/4508058220/Code+Guidelines+-+React+Native#Folder-organization